### ISSUE TITLE
Recategorise some tools for genome annotation

### ISSUE THREAD

**Issue by erasche** [galaxyproject/tools-iuc#2593]
@bgruening @abretaud 

We should update the following to include GA in their .shed.yml:

- interproscan
- all GGA tools
- maker / busco 
- augustus / glimmer / any other gene callers
- circos? maybe?
- antiSMASH
- hmmer / blast? these are commonly used there?
- staramr
- progressiveMauve / lastz

**Comment by bgruening** [comment]
Good idea. I propose the following, we create PRs with [ci skip] and update the categories also by hand in the TS.

**Comment by erasche** [comment]
:+1:
